Feat frunphi

jtwhite79 opened the pull request:
added super hack to get weighted phi as an observation.  refactored draw_new to be at the `Ensemble` level with derived methods for `ParameterEnsemble` and `ObservationEnsemble`.  Some small mods to `pars_as_obs()` so that user-tag can be added to the resulting obs names.

jtwhite79 committed:
added twisted function to calculate phi as a model post processor so that we can have phi as an observation in sqp testing

jtwhite79 committed:
fix

jtwhite79 committed:
some mods to pars_as_obs

jtwhite79 committed:
fix

jtwhite79 committed:
mod to add transform columns to exclude offset and scale

jtwhite79 committed:
Merge branch 'develop' of https://github.com/pypest/pyemu into feat_frunphi

jtwhite79 committed:
moved draw new under Ensemble with derived methods in par and obs en

jtwhite79 committed:
added test for draw new obs en

briochh commented on pyemu/en.py:
```diff
@@ -833,6 +909,34 @@ def from_gaussian_draw(
+                under-varied new realizations resulting from npar >> nreals in `self`. If True,
+                The standard devation is set to one over the square root on number of reals in 
+                `self`.  
+            noise_reals (ParameterEnsemble): other existing realizations (likely prior realizations)
```
should this be ObservationEnsemble now?

briochh commented on pyemu/en.py:
```diff
@@ -833,6 +909,34 @@ def from_gaussian_draw(
+                is True and `noise_reals` is None.
+        
+        Returns
+            ParameterEnsemble
```
ObservationEnsemble?

briochh committed:
Update en.py

Switcheroo ParameterEnsemble for ObservationEnsemble in child draw new doc strings.

briochh committed:
Merge branch 'develop' into feat_frunphi
